lib/ui/insta_upload_photo_screen.dart

```diff
@@ -10,7 +10,7 @@ import 'package:image/image.dart' as Im;
 import 'package:path_provider/path_provider.dart';
 import 'dart:math';
 
-class InstaUploadPhotoScreen extends StatefulWidget {
+class InstaUploadPhotoScreen extends StatefulWidget {  //immutable class again no final to explore ?
```

Comment:
Here, immutable class is required as well as <b>imageFile</b> cannot be <b>final</b> because in <b>compressImage()</b> method, it is being provided a new value